query-algebrizer/src/cc.rs

```diff
@@ -115,13 +128,29 @@ pub fn default_table_aliaser() -> TableAliaser {
     })
 }
 
-#[derive(PartialEq, Eq, Debug)]
+#[derive(PartialEq, Eq)]
 pub enum ColumnConstraint {
     EqualsEntity(QualifiedAlias, Entid),
     EqualsValue(QualifiedAlias, TypedValue),
     EqualsColumn(QualifiedAlias, QualifiedAlias),
 }
 
+impl fmt::Debug for ColumnConstraint {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match self {
+            &ColumnConstraint::EqualsEntity(ref qa, ref entid) => {
+                write!(f, "cc::Entity({}/{:?} = {:?})", qa.0, qa.1, entid)
```

Comment:
If you really want these to look like standalone delimited things, wrap them in `[]`, perhaps? No preference from me.